test/v3/removeLiquidity.integration.test.ts

```diff
@@ -419,3 +467,47 @@ class MockApi {
 }
 
 /******************************************************************************/
+
+/*********************** Helper functions for this test ***********************/
+async function putPoolIntoRecoveryMode(
+    client: any,
+    poolState: PoolState,
+    authorizedAddress: Address,
+) {
+    // grant the testAddress the right to enable Recovery mode for pools
+    const { request: grantRoleRequest } = await client.simulateContract({
+        address: AUTHORIZER[chainId],
+        abi: parseAbi(['function grantRole(bytes32 role, address account)']),
```

Comment:
You might consider importing and using [the authroizer ABI](https://github.com/balancer/b-sdk/blob/feat/remove-liq-recovery/src/abi/authorizer.ts) instead of using `parseAbi` since all the other `client.simulateContract`s are using the import abi pattern

I'm somewhat indifferent cus it is nice for readability to have the func sig visible right next to the args below, but it does violate DRY principle a bit

Reply:
Good suggestion. I implemented your approach.